### ISSUE TITLE
Filemanager does not show preview for webp format

### ISSUE THREAD

**Issue by meshkov** [modxcms/revolution#15036]
### Summary
Filemanager does not show preview for webp format

### Step to reproduce
Try to uploud webp files.

### Observed behavior
![image](https://user-images.githubusercontent.com/1703670/77632846-a049f880-6f5f-11ea-8a94-c80638c9bc00.png)

### Expected behavior
Need preview for webp format. Very hard to work without the ability to preview the file

### Environment
MODX 2.7.3 and MODX 3.0

**Comment by meshkov** [comment]
MODX 2.7.3 media browser:
![image](https://user-images.githubusercontent.com/1703670/77636546-56fca780-6f65-11ea-9605-24f535bcdc79.png)

MODX3 In media browser its work:
![image](https://user-images.githubusercontent.com/1703670/77636408-1bfa7400-6f65-11ea-8d3f-18accc49ac87.png)
In MODX 3 problem only in filemanager

**Comment by verstratenbram** [comment]
I solved this by editing the `ImageCreateFromFilename()` function in `core/model/phpthumb/phpthumb.class.php`:
```
$ImageCreateFromFunction = array(
	1  => 'imagecreatefromgif',
	2  => 'imagecreatefromjpeg',
	3  => 'imagecreatefrompng',
	15 => 'imagecreatefromwbmp',
	18 => 'imagecreatefromwebp', // <<ADD THIS LINE>>
);
```
```
switch (@$getimagesizeinfo[2]) {
	case 1:  // GIF
	case 2:  // JPEG
	case 3:  // PNG
	case 15: // WBMP
	case 18: // WEBP // <<ADD THIS LINE>>
		$ImageCreateFromFunctionName = $ImageCreateFromFunction[$getimagesizeinfo[2]];
		if (function_exists($ImageCreateFromFunctionName)) {
			$this->DebugMessage('Calling '.$ImageCreateFromFunctionName.'('.$filename.')', __FILE__, __LINE__);
			$ImageCreateWasAttempted = true;
			$gd_image = $ImageCreateFromFunctionName($filename);
		} else {
			$this->DebugMessage('NOT calling '.$ImageCreateFromFunctionName.'('.$filename.') because !function_exists('.$ImageCreateFromFunctionName.')', __FILE__, __LINE__);
		}
		break;
```

**Comment by Jako** [comment]
@verstratenbram Please create a PR with your changes on the original repository: https://github.com/JamesHeinrich/phpThumb

**Comment by verstratenbram** [comment]
The PR has been created [https://github.com/JamesHeinrich/phpThumb/pull/181](https://github.com/JamesHeinrich/phpThumb/pull/181)

**Comment by Ibochkarev** [comment]
Fixed #15816
